utils/wrappers/rebalancingWrapper.ts

```diff
@@ -495,4 +498,131 @@ export class RebalancingWrapper {
       priceDenominator,
     };
   }
+
+  /* ============ Rebalancing Token Manager ============ */
+
+  public async deployRebalancingTokenManagerAsync(
+    coreAddress: Address,
+    btcPriceFeedAddress: Address,
+    ethPriceFeedAddress: Address,
+    btcAddress: Address,
+    ethAddress: Address,
+    setTokenFactoryAddress: Address,
+    auctionLibrary: Address,
+    auctionTimeToPivot: BigNumber = new BigNumber(100000),
+    from: Address = this._tokenOwnerAddress
+  ): Promise<RebalancingTokenManagerContract> {
+    const truffleRebalacingTokenManager = await RebalancingTokenManager.new(
+      coreAddress,
+      btcPriceFeedAddress,
+      ethPriceFeedAddress,
+      btcAddress,
+      ethAddress,
+      setTokenFactoryAddress,
+      auctionLibrary,
+      auctionTimeToPivot,
+      { from },
+    );
+
+    return new RebalancingTokenManagerContract(
+      new web3.eth.Contract(truffleRebalacingTokenManager.abi, truffleRebalacingTokenManager.address),
+      { from, gas: DEFAULT_GAS },
+    );
+  }
+
+  public getExpectedNextSetParameters(
+    btcPrice: BigNumber,
+    ethPrice: BigNumber,
+  ): any {
+    let units: BigNumber[];
+    let naturalUnit: BigNumber;
+    if (btcPrice.greaterThanOrEqualTo(ethPrice)) {
+      const ethUnits = btcPrice.mul(new BigNumber(10 ** 10)).div(ethPrice).round(0, 3);
+      units = [new BigNumber(1), ethUnits];
+      naturalUnit = new BigNumber(10 ** 10);
```

Comment:
Feels like `new BigNumber(10 ** 10)` could be a constant. `DEFAULT_SET_TOKEN_NATURAL_UNIT`